lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # `context` block descriptions should start with 'when', or 'with'.
+      #
+      # @see https://github.com/reachlocal/rspec-style-guide#context-descriptions
+      # @see http://www.betterspecs.org/#contexts
+      #
+      # @example `Prefixes` configuration option, defaults: 'when', and 'with'
+      #   Prefixes:
+      #     - when
+      #     - with
+      #     - without
+      #
+      # @example
+      #   # bad
+      #   context 'the display name not present' do
+      #     ...
+      #   end
+      #
+      #   # good
+      #   context 'when the display name is not present' do
+      #     ...
+      #   end
+      class ContextWording < Cop
+        MSG = 'Start context description with %s.'.freeze
+
+        def_node_matcher :context_wording, <<-PATTERN
+          (block (send _ { :context :shared_context } $(str #bad_prefix?)) ...)
+        PATTERN
+
+        def on_block(node)
+          context_wording(node) do |context|
+            add_offense(context, message: message)
+          end
+        end
+
+        private
+
+        def bad_prefix?(description)
+          !prefixes.include?(description.partition(' ').first)
```

Comment:
You probably want to use `#split` over `#partition`, as the latter will add the match to the result. `String#split` also defaults to splitting on spaces. 🙂

Reply:
```ruby
" foo bar".split
#=> ["foo", "bar"]

" foo bar".partition(" ")
#=> ["", " ", "foo bar"]
```

Reply:
It's so tempting to switch it back to `partition` now when you mention this.
If it's worth registering an offense for descriptions starting from whitespace as well.